### PR TITLE
primordial sentinel: gas grenade and greyscale xeno grenade support

### PR DESCRIPTION
**Pull request by TiviPlus**
Gas grenade which sentinel throws then makes a 3 tile radius of neuro smoke, sprites from CM

### CODE REVIEW

**Review by Kuro020 — APPROVED** [review]

File: code/modules/mob/living/carbon/xenomorph/castes/sentinel/abilities_sentinel.dm [hunk 1]
```diff
@@ -1 +1,37 @@
...
+	name = "Throw neurogas grenade"
+	action_icon_state = "gas mine"
+	mechanics_text = "Throws a gas emitting grenade at your enemies."
+	keybind_signal = COMSIG_XENOABILITY_NEUROGAS_GRENADE
+	plasma_cost = 300
+	cooldown_timer = 1 MINUTES
+
+/datum/action/xeno_action/activable/neurogas_grenade/use_ability(atom/A)
+	. = ..()
+	var/turf/T = get_turf(owner)
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/castes/sentinel/abilities_sentinel.dm:12]: useless var

File: code/modules/projectiles/guns/specialist.dm [hunk 2]
```diff
@@ -822,7 +822,6 @@
 		log_explosion("[key_name(user)] fired a grenade [F] from \a [src] at [AREACOORD(user.loc)].")
 		F.icon_state = initial(F.icon_state) + "_active"
 		F.activate(user)
-		F.updateicon()
```
Comment (BraveMole) [inline code/modules/projectiles/guns/specialist.dm:825]: you tested that this is not making any change?

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/specialist.dm (hunk 2 above)
Comment (TiviPlus) [reply code/modules/projectiles/guns/specialist.dm:825]: tested on both standard and this nade

File: code/modules/mob/living/carbon/xenomorph/castes/sentinel/abilities_sentinel.dm (hunk 1 above)
Comment (TiviPlus) [reply code/modules/mob/living/carbon/xenomorph/castes/sentinel/abilities_sentinel.dm:12]: changing

Comment (TiviPlus) [reply code/modules/mob/living/carbon/xenomorph/castes/sentinel/abilities_sentinel.dm]: done

**Review by BraveMole — APPROVED** [review]

**Pull request merged**